docs/guides/mysql/clustering/read-replica/index.md

```diff
@@ -0,0 +1,358 @@
+menu:
+  docs_{{ .version }}:
+    identifier: guides-mysql-clustering-read-replica
+    name: MySQL Read Replica Guide
+    parent: guides-mysql-clustering
+    weight: 21
+menu_name: docs_{{ .version }}
+section_menu_id: guides
+---
+
+> New to KubeDB? Please start [here](/docs/README.md).
+
+# KubeDB - MySQL Read Replica
+
+This tutorial will show you how to use KubeDB to provision a MySQL Read Replica from a kubedb managed mysql instance.
+
+## Before You Begin
+
+Before proceeding:
+
+- Read [mysql replication concept](/docs/guides/mysql/clustering/overview/index.md) to learn about MySQL Replication.
+
+- You need to have a Kubernetes cluster, and the kubectl command-line tool must be configured to communicate with your cluster. If you do not already have a cluster, you can create one by using [kind](https://kind.sigs.k8s.io/docs/user/quick-start/).
+
+- Now, install KubeDB cli on your workstation and KubeDB operator in your cluster following the steps [here](/docs/setup/README.md).
+
+- To keep things isolated, this tutorial uses a separate namespace called `demo` throughout this tutorial. Run the following command to prepare your cluster for this tutorial:
+
+```bash
+  $ kubectl create ns demo
+  namespace/demo created
+  ```
+
+> Note: The yaml files used in this tutorial are stored in [docs/guides/mysql/clustering/read-replica/yamls](https://github.com/kubedb/docs/tree/{{< param "info.version" >}}/docs/guides/mysql/clustering/group-replication/yamls) folder in GitHub repository [kubedb/docs](https://github.com/kubedb/docs).
+## Read Replica
+
+Read Replica allows us to replicate data from one mysql source to a read-only mysql server. In this section we will provision a mysql server with kubedb, and then we will create a read replica from it.
+
+
+## Deploy Mysql server
+
+The following is an example `MySQL` object which creates a MySQL standalone instance
+
+```yaml
+apiVersion: kubedb.com/v1alpha2
+kind: MySQL
+metadata:
+  name: mysql
+  namespace: demo
+spec:
+  allowedReadReplicas:
+      namespaces:
+        from: Same
+      selector:
+        matchLabels:
+          kubedb.com/instance_name: ReadReplica  
+  version: "8.0.29"
+  storageType: Durable
+  storage:
+    storageClassName: "standard"
+    accessModes:
+      - ReadWriteOnce
+    resources:
+      requests:
+        storage: 1Gi
+  terminationPolicy: WipeOut
+```
+
+```bash
+$ kubectl create -f https://github.com/kubedb/docs/raw/{{< param "info.version" >}}/docs/guides/mysql/clustering/read-replica/yamls/mysql.yaml
+mysql.kubedb.com/mysql created
+```
+Here,
+
+ - `spec.AllowReadReplicas`  defines the types of read replicas that MAY be attached to a MySQL instance and the trusted namespaces where those Read Replica resources MAY be present.You will be able to set namespace `spec.allowReadReplicas.NameSpace` and labels `spec.allowReadReplicas.selector`.For more see [here](https://github.com/kubedb/apimachinery/blob/master/apis/kubedb/v1alpha2/mysql_types.go#L159).
```

Comment:
In this part `replicas that MAY be attached`, why maybe has `MAY` in capital and separate?